src/components/CowPen/CowPen.test.js

```diff
@@ -40,19 +42,19 @@ describe('Cow', () => {
 
   describe('movement', () => {
     test('schedules a position change at boot', () => {
-      expect(setTimeout).toHaveBeenCalledWith(
+      expect(global.setTimeout).toHaveBeenCalledWith(
```

Comment:
even though you use `global` for the spy do you actually have to change these references? i think they are references to the global versions by default, but i'm not positive

Reply:
Yeah, this change was necessary to fix some newly-breaking tests. It seems that newer versions of Jest have changed how timers work. Accessing via `global` is the only way to spy on them as necessary for these tests (that I've found so far).